src/Microsoft.IdentityModel.Protocol.Extensions/Microsoft.IdentityModel.Protocols.xproj

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We have .kproj in other branches. we had .xproj earlier for a while in one of the VS update but now it's .kproj again.